cs/test/core/ConvertTests.cs

```diff
@@ -30,6 +79,12 @@ public class Decimals
         [global::Bond.Id(5), global::Bond.Type(typeof(global::Bond.Tag.blob)), global::Bond.Required]
         public decimal _decRequired { get; set; }
 
+        [global::Bond.Id(6), global::Bond.Type(typeof(global::Bond.Tag.blob)), global::Bond.RequiredOptional]
+        public decimal _decRequiredOptional { get; set; }
+
+        [global::Bond.Id(7), global::Bond.Type(typeof(global::Bond.Tag.blob))]
+        public decimal _decOptional { get; set; }
```

Comment:
`_decOptional` looks to be the same as `_dec`, so one of them can be removed. Same thing for `_date` and `_ref`.